app.py

```diff
@@ -1,11 +1,21 @@
 from flask import Flask, request
 import sqlite3
-
 from serializers import serialize_book
 
 app = Flask(__name__)
 
 
+def put_update_book(connection, book_id):  # PUT request
+    body = request.json
+    new_book_name = body['name']
+    if new_book_name == '':
+        return {'error': 'Book name cannot be empty'}, 400
+    cursor = connection.cursor()
+    cursor.execute(f"update books set name = '{new_book_name}' where id = {book_id}")
```

Comment:
pls add validation that the book with the book_id, you passed in the request exists.